decidim-admin/app/helpers/decidim/admin/attributes_display_helper.rb

```diff
@@ -31,7 +31,14 @@ module AttributesDisplayHelper
       # attrs - a list of N attributes of the `record`.
       def display_for(record, *attrs)
         attrs.map do |attr|
-          display_label(record, attr) + display_value(record, attr)
+          if record.column_for_attribute(attr).type == :hstore
```

Comment:
This is kinda weak, isn't there any way to introspect if an attribute is translatable?